web/src/views/FileBrowserView/FileBrowser.vue

```diff
@@ -461,7 +482,10 @@ async function getItems() {
       // Inject relative path
       name: path.path.split('/').pop()!,
       // Inject services
-      services: getExternalServices(path) || undefined,
+      services: getExternalServices(path, {
```

Comment:
disregard this comment -- just testing if I have superpowers
```suggestion
      services: getExternalServices(DISREGARD, {
```